backend/config/settings.py

```diff
@@ -2,6 +2,7 @@
 
 from pydantic import BaseSettings
 
+

```

Comment:
@devvspaces please avoid touching files that are not related to the work you are doing. 8 files changes for this task gives me a lot of worry